### ISSUE TITLE
[Airtable] find and update records

### ISSUE THREAD

**Issue by mspych** [raycast/extensions#14773]
### Extension

https://www.raycast.com/marks/raycast-airtable-extension

### Description

The current Raycast Airtable extension allows viewing and listing bases but lacks functionality for directly interacting with records. I propose adding features that enable users to search for specific records by name and update them directly within the extension. 
Not sure how this could work in details, but maybe it would be possible to configure some fields that are editable in the configuration.
Achieving functionality like [this one](https://youtu.be/m5MDv9qwhU8?t=281) would be beyond amazing. 
This enhancement will greatly improve efficiency by minimizing the need to switch to the base or interface.

**Comment by marks** [comment]
Chiming real quick to say that I agree this would be a great feature set. I dont think I'll be able to take it on in the near future, so any other developer should definitely consider it!

**Comment by xmok** [comment]
I took a very quick stab at this:

https://github.com/user-attachments/assets/86f19a0c-c4b8-47ca-89f8-2a5b26367e76

In the above screencast we are now able to View/List records in a base - there is no editing as of yet because will have to think of an efficient way to handle it. In the meantime, I can at least make a PR for the above screencast but we will need to modify the OAuth application in Airtable to ask for the additional scope of `data.records:read`. Presently I have created my own Dev OAuth Integration for the screencast.
